docs/source/json-api/index.rst

```diff
@@ -34,9 +35,10 @@ We welcome feedback about the JSON API on
 
    lf-value-specification
    search-query-language
+   http-status-codes
 
-Running the JSON API
-********************
+Running the JSON API Service
```

Comment:
Let’s please not introduce yet another name and stick with `JSON API`.

Reply:
It's called "HTTP JSON API Service" in the header/section name as well. Should I change this to "JSON API"?